### PR TITLE
Gradient Boosting: translation (si)

### PR DESCRIPTION
**Pull request by anjamejac**
Updated non-translated Gradient Boosting (si).

### CODE REVIEW

File: si/orange3.jaml [hunk 1]
```diff
@@ -2414,7 +2414,7 @@ modelling/base.py:
 modelling/catgb.py:
     CatGBLearner: false
     class `CatGBLearner`:
-        Gradient Boosting (catboost): false
+        Gradient Boosting (catboost): Gradientno povečanje (catboost)
```
Comment (janezd) [inline si/orange3.jaml:2417]: Za tale prevod (boosting -> povečanje) nisem prepričan. Mogoče za vsak slučaj vprašaš @markotoplak in @BlazZupan?

File: si/orange3.jaml [hunk 2]
```diff
@@ -9413,113 +9413,106 @@ widgets/model/owcurvefit.py:
...
-            'Limit depth of individual trees: ': null
-            Subsampling: null
+            callback: false
+            alignment: false
+            controlWidth: false
+            Basic Properties: Osnovne lastnosti
+            n_estimators: false
+            Number of trees:: Število dreves:
+            learning_rate: false
+            'Learning rate: ': 'Stopnja učenja: '
```
Comment (janezd) [inline si/orange3.jaml:9433]: Stopnja
